.github/workflows/linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  lighthouse:
```

Comment:
- [ ] I suggest you follow the [guide](https://github.com/microverseinc/linters-config/tree/master/html-css) to set up the .hintrc and .stylintrc.json linter files. This will help you to catch any hidden bugs locally.